libafl/src/bolts/shmem.rs

```diff
@@ -1235,16 +1236,17 @@ pub mod win32_shmem {
                 let map_str_bytes = id.id;
                 // Unlike MapViewOfFile this one needs u32
                 let handle = OpenFileMappingA(
-                    FILE_MAP_ALL_ACCESS,
+                    FILE_MAP_ALL_ACCESS.0,
                     BOOL(0),
-                    PSTR(map_str_bytes.as_ptr() as *mut _),
-                );
-                if handle == HANDLE(0) {
-                    return Err(Error::unknown(format!(
+                    PCSTR(map_str_bytes.as_ptr() as *mut _),
+                )
+                .map_err(|_| {
```

Comment:
Why not take the error message of the original error?
Also, you could add a From<..> for this Error to libAFL if you wanted

Reply:
ok